### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by Byeclouds**
@Rinnki97
@AlexZhang21
@Liu1xuan

### CODE REVIEW

File: prac_06/guitar_test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from guitar import Guitar
+
+
+def main():
+    guitar1 = Guitar("Gibson L-5 CES", 1922, 16035.40)
+    guitar2 = Guitar("Another Guitar", 2013, 1500.00)
+
+    # Test get_age() method
+    expected_age1 = 100
+    age1 = guitar1.get_age()
```
Comment (AlexZhang21) [inline prac_06/guitar_test.py:10]: Consider edge cases or invalid inputs.
